core/remote_file.py

```diff
@@ -52,18 +52,23 @@ def ssh_pub_key(self):
         pub_keys = glob.glob(os.path.join(ssh_dir, '*.pub'))
         return pub_keys[0]
 
-    def connect_ssh(self):
+    def connect_ssh(self, use_gssapi):
         import paramiko
         ssh = paramiko.SSHClient()
         ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
 
         try:
-            # Login server with ssh public key.
-            pub_key = self.ssh_pub_key()
-            ssh.connect(self.ssh_host, port=self.ssh_port, username=self.ssh_user, key_filename=pub_key)
+            if use_gssapi:
+                import gssapi
```

Comment:
@werhner I got error from ruff "gssapi imported but unused`, we should remove this line?

Reply:
It's ok to remote this line.